doc/content/getting-started/ttn/addresses.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Addresses"
+description: ""
+---
+
+The addresses where you can reach The Things Stack Community consist of the cluster ID and `cloud.thethings.network`.
+
+You can update the examples below with your cluster ID by filling it here.
+
+{{< cluster-selector >}}
+
+## Console
+
+To learn how to get started with the Console, see the [Console Getting Started Guide]({{< ref "/getting-started/console" >}}).
+
+Each cluster has a Console where you can manage gateways and end devices of that cluster. You can use any cluster with your Cloud account.
+
+The address of the Console includes the cluster ID:
```

Comment:
```suggestion
The address where you can access the Console:
```

Reply:
The info that it contains the cluster ID is already mentioned on the beginning, and it's better to keep it as simple as possible